nhais/main.py

```diff
@@ -0,0 +1,50 @@
+import tornado.httpclient
+import utilities.integration_adaptors_logger as log
+from requests import healthcheck_handler
+from utilities import config
+
+logger = log.IntegrationAdaptorsLogger(__name__)
+
+def configure_http_client():
+    """
+    Configure Tornado to use the curl HTTP client.
+    """
+    tornado.httpclient.AsyncHTTPClient.configure('tornado.curl_httpclient.CurlAsyncHTTPClient')
+
+def start_tornado_server() -> None:
+
+    supplier_application = tornado.web.Application(
+        [
+            (r"/healthcheck", healthcheck_handler.HealthcheckHandler)
+        ])
+    supplier_server = tornado.httpserver.HTTPServer(supplier_application)
+    server_port = int(config.get_config('SERVER_PORT', default='90'))
```

Comment:
Why use 90 as default for HTTP server?

Reply:
90 doesn't work, changed to 80 to match example env file